plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -121,7 +121,7 @@ public DistributedQueryRunner build()
     public static DistributedQueryRunner createDeltaLakeQueryRunner(String catalogName)
             throws Exception
     {
-        return createDeltaLakeQueryRunner(catalogName, ImmutableMap.of(), ImmutableMap.of());
+        return createDeltaLakeQueryRunner(catalogName, ImmutableMap.of(), ImmutableMap.of("delta.register-table-procedure.enabled", "true"));
```

Comment:
Revert -- we don't need to enable this by default for all the tests.

Reply:
Reverted